src/doc/trpl/hello-cargo.md

```diff
@@ -73,7 +74,8 @@ extra goodies. According to the TOML docs,
 
 [toml]: https://github.com/toml-lang/toml
 
-Once you have this file in place, we should be ready to build! To do so, run:
+Once you have this file in place in your project's root directory, we should be
```

Comment:
There are still a lot of "you" and "your" in the file. What is the rule for "us" vs "you"?